webapp/src/components/Analysis/UtterancesTable.tsx

```diff
@@ -390,6 +399,37 @@ const UtterancesTable: React.FC<Props> = ({
   ];
 
   const searchString = constructSearchString(pipeline);
+
+  const importProposedActions = (file: File) => {
+    const fileReader = new FileReader();
+    fileReader.onload = ({ target }) => {
+      if (target) {
+        const result = target.result as string;
+        const [header, ...rows] = result.split(/\r?\n/).slice(0, -1);
+        if (rows.length === 0) {
+          raiseErrorToast("There are no records in the CSV file.");
+          return null;
+        }
+        if (header !== `${config.columns.persistent_id},proposed_action`) {
+          raiseErrorToast(
+            `The CSV file must have column headers ${config.columns.persistent_id} and proposed_action, in that order.`
+          );
+          return null;
```

Comment:
`fileReader.onload` is not supposed to return anything, so I would remove those `null`s.
```suggestion
          return;
        }
        if (header !== `${config.columns.persistent_id},proposed_action`) {
          raiseErrorToast(
            `The CSV file must have column headers ${config.columns.persistent_id} and proposed_action, in that order.`
          );
          return;
```